frontend/src/components/CustomTableForm.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div>
+    <div class="form-class-text">
+      <h4 style="padding-left: 30px"> You can start creating your custom table here by checking the UL campaigns you're following: </h4>
+      <div class="form-class" style="text-align: center">
+        <span class="input-class"  v-for="campaign in existingCampaigns" :key="campaign.name"><label for="campaign.name">{{campaign.name}}</label>
+        <input type="checkbox" v-on:change="generateURL" v-bind:id="campaign.name" v-bind:value="campaign.name" v-model="campaigns"></span>
+      </div>
+  
+    <h4 style="padding-left: 30px"><label for="datasetnames">Please enter the dataset names you want to follow in your table separated by a | below with no spaces: </label></h4>
+    <input style="text-align: center; border: 1px solid black; width: 60%;background-color: white;border-style: solid;" v-model="datasetnames" v-on:input="generateURL" id="datasetnames" placeholder="TTToHadronic_|ST_t-channel_top_4f_">
```

Comment:
`border-style: solid;` is redundant, `solid` is already specified in `border: 1px solid black`